frontend/lib/ui/InputGrid/InputGrid.module.css

```diff
@@ -1,17 +1,21 @@
 .inputgrid {
   margin-top: 1rem;
   width: 100%;
-  border-spacing: 0;
```

Comment:
Looks like removing this caused the cell padding as mentioned in https://github.com/kiesraad/abacus/pull/255#issuecomment-2285385684.

Reply:
Yes I re-added it, so weird because the browser said this had no effect XD